Python/cp/longest_Common_Prefix.py

```diff
@@ -0,0 +1,56 @@
+# Python Program to find the longest common prefix
+# function to find minimum length
+
+
+def findMinLength(string, size):
```

Comment:
@narayan2111  I am getting an error of `IndexError: list index out of range ` in this function. Can you please fix this?

Reply:
> @narayan2111 I am getting an error of `IndexError: list index out of range ` in this function. Can you please fix this?

Yes Ma'am , I fixed this error

Reply:
> @narayan2111 I am getting an error of `IndexError: list index out of range ` in this function. Can you please fix this?

Yes ma'am I fixed the error. @Kajol-Kumari